### ISSUE TITLE
[New Rule] Simplify "in dict.keys()" to "in dict"

### ISSUE THREAD

**Issue by Skylion007** [MartinThoma/flake8-simplify#40]
## Explanation

Most of the discussion of the motivation about this rule can be found in this very detailed [StackOverflow post](https://stackoverflow.com/a/29314342)

## Example

```python
# Bad
key in dict.keys()

# Good
key in dict
```

Basically, the later way of doing this pythonic and slightly more performant and arguably more Readable. The previous method only exists for legacy code (<= python 2.2). So there is no reason to use the former in modern codebases.

**Comment by MartinThoma** [comment]
```
$ astpretty --no-show-offsets /dev/stdin <<< `cat example.txt`
Module(
    body=[
        Expr(
            value=Compare(
                left=Name(id='key', ctx=Load()),
                ops=[In()],
                comparators=[
                    Call(
                        func=Attribute(
                            value=Name(id='dict', ctx=Load()),
                            attr='keys',
                            ctx=Load(),
                        ),
                        args=[],
                        keywords=[],
                    ),
                ],
            ),
        ),
    ],
    type_ignores=[],
)
```
